ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+Please try to adhere to the guidelines below as much as possible when submitting your issue.
+
+If you are submitting a bug issue:
+- [ ] Please include your operating system type and version number, as well as your Python, scikit-learn, numpy, and scipy versions. This information can be found by runnning [this code snippet](https://gist.github.com/nelson-liu/108ad4a0571e43ac3d80).
```

Comment:
@jakevdp that sounds like a great idea, but I'm unsure if it should go in sklearn/__init__.py or somewhere else like the sklearn/utils. Thoughts?
Also -- how would you check whether the installation was done via wheel, conda, etc. Module path?